src/Library/CopyJob.php

```diff
@@ -34,7 +34,7 @@ final class CopyJob
     protected array $options = [
         'override' => false,
         'delete' => false,
-        'merge' => 'preserve',
```

Comment:
@markocupic  Preserve will actually merge but preserve existing keys in the file. See readme. 
Maybe this option needs to be renamed to something more explicit?
'keep-keys'?
and change the other one to 'replace keys'?

Reply:
thanks for the review, I have added 1 comment